In [ ]:
import statsmodels.api as sm

views_a = 2000
views_b = 2500
purchases_a = 220
purchases_b = 240

z_stat, p_value = sm.stats.proportions_ztest([purchases_a, purchases_b], [views_a, views_b])

print(f"Z-statistic: {z_stat}")
print(f"P-value: {p_value}")

if p_value < 0.05:
    print("існує статистично значуща різниця між кампаніями. A and B.")
else:
    print("Не існує статистично значуща різниця між кампаніями A and B.")

Z-statistic: 1.540457412600049
P-value: 0.12344889604606361
Не існує статистично значуща різниця між кампаніями A and B.


P-value: 0.123 > 0.05, ми не маємо статистичних підстав відхиляти нульову гіпотезу. Це означає, що різниця у конверсії між версіями кампаній A та B не є статистично значущою на рівні довіри 95%.

З точки зору статистики, ми не можемо стверджувати, що одна кампанія працює краще за іншу — їхні результати еквівалентні з точки зору конверсії.